### PR TITLE
Create padStart.md

### PR DESCRIPTION
**Pull request by AlstrupJacob**
### Description

created padStart.md. resolves issue #3729

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/strings/padStart/padStart.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Tags: 
+  - 'String'
+  - 'Padding'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.padStart()`** method will pad the start of the sequence with the desired fill character to the specified length, returning the new character seuqence.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/padStart/padStart.md:16]:
```suggestion
The **`.padStart()`** method will pad the start of the sequence with the desired fill character to the specified length, returning the new character sequence. **`.padStart()`** is also used for left padding. 
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side.

**Comment by AlstrupJacob** [comment]
@dakshdeepHERE done

**Comment by dakshdeepHERE** [comment]
image
@AlstrupJacob where are the requested changes applied? The syntax is still the same.

**Comment by AlstrupJacob** [comment]
> image @AlstrupJacob where are the requested changes applied? The syntax is still the same.

very strange, are they there now?

**Comment by dakshdeepHERE** [comment]
> > image @AlstrupJacob where are the requested changes applied? The syntax is still the same.
> 
> very strange, are they there now?

Let me check. If they are not there I will tell you how to commit these changes easily.

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Syntax
+
+```pseudo
+String.padStart(length: Int, padChar: String): String
+```
+
+`length`: The amount of padding to be added, in number of characters.
+`padChar`: Character to be used as padding, defaults to ' '.
+
+## Examples
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:27]:
```suggestion
## Example
```
One change left.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]

**Comment by AlstrupJacob** [comment]
done sorry again

**Comment by dakshdeepHERE** [comment]
@AlstrupJacob commit this last change and I will do the overall review and move to 2nd review part.

**Comment by dakshdeepHERE** [comment]
> done sorry again

It's okay. :)

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+The **`.padStart()`** method will pad the start of the sequence with the desired fill character to the specified length, returning the new character sequence. **`.padStart()`** is also used for left padding.
+
+## Syntax
+
+```pseudo
+String.padStart(length: Int, padChar: String): String
+```
+
+`length`: The amount of padding to be added, in number of characters.
+`padChar`: Character to be used as padding, defaults to ' '.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:25]:
```suggestion
- `length`: The amount of padding to be added, in number of characters.
- `padChar`: Character to be used as padding, defaults to ' '.
```

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+```kotlin
+fun main {
+  val str = "codecademydocs"
+
+  val str2 = str.padStart(3)
+  println(str2)
+
+  val str3 = str.padStart(4, "-")
+  println(str3) 
+  }
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:40]:
```suggestion
fun main() {
  val str = "codecademydocs"

  val str2 = str.padStart(14)
  println(str2)

  val str3 = str.padStart(18, '-')
  println(str3)
}
```
The print the below output you'll have to do these changes. The length of the string is 14. If you use 3 and 4 the string will be printed. No padding will be there.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Last two changes

**Comment by dakshdeepHERE** [comment]
@AlstrupJacob Last 2 changes, then we will move for 2nd review.

**Comment by AlstrupJacob** [comment]
> @AlstrupJacob Last 2 changes, then we will move for 2nd review.

done

**Review by dakshdeepHERE — APPROVED** [review]

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+---
+Title: '.padStart()'
+Description: 'Pads the start of a character sequence with a string of the specified character and length.'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:3]:
```suggestion
Description: 'Pads the start of a string with a specified character.' 
```

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.padStart()`** method pada the begining of the sequence with the specified fill character to reach the desired length, resulting in a new character sequence. **`.padStart()`** is also used for left-padding.
+
+## Syntax
+
+```pseudo
+String.padStart(length: Int, padChar: String): String
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:21]:
```suggestion
String.padStart(length, padChar)
```

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+-`length`: The amount of padding to be added, in number of characters.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:24]:
```suggestion
-`length`: An integer that represents the total length of the string returned (with padding).
```

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+-`padChar`: Character to be used as padding, defaults to ' '.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:25]:
```suggestion
-`padChar`: The character to be used (enclosed by `''`) as padding, defaults to whitespace.
```

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+The **`.padStart()`** method pada the begining of the sequence with the specified fill character to reach the desired length, resulting in a new character sequence. **`.padStart()`** is also used for left-padding.
+
+## Syntax
+
+```pseudo
+String.padStart(length: Int, padChar: String): String
+```
+
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:23]:
```suggestion

- 'String': The string to be modified.
```

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+-`length`: The amount of padding to be added, in number of characters.
+-`padChar`: Character to be used as padding, defaults to ' '.
+
+## Example
+
+The example demonstrates the use of `.padStart()` to pad a string.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:29]:
```suggestion
The example below demonstrates the use of `.padStart()` to pad a string.
```

File: content/kotlin/concepts/strings/terms/padStart/padStart.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+```kotlin
+fun main() {
+  val str = "codecademydocs"
+
+  val str2 = str.padStart(14)
+  println(str2)
+
+  val str3 = str.padStart(18, "-")
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/padStart/padStart.md:38]:
```suggestion
  val str3 = str.padStart(18, '-')
```

**Comment by AlstrupJacob** [comment]
@caupolicandiaz done

**Review by caupolicandiaz — APPROVED** [review]
@AlstrupJacob Looks good

**Pull request merged**